Sprint-1/implement/dedupe.js

```diff
@@ -1 +1,19 @@
-function dedupe() {}
+function dedupe(arr) {
+     const removeRepeated = arr.filter((element, index) => {
+       // console.log("element >>", element, "index >>", index) 
+       
+        return arr.indexOf(element) === index
+     })
+   console.log(removeRepeated)
```

Comment:
This statement should not belong inside the function. 

Please note that the way you construct `removeRepeated` works, but this "algorithm" will perform poorly on huge array because `arr.indexOf(element)` needs to search for `element` in the array from the start of the array for every element in the array.